app/javascript/components/App.js

```diff
@@ -8,14 +8,18 @@ import mockquestions from "./mockquestions.js";
 import TriviaShow from "./pages/TriviaShow";
 // import Header from './components/Header.js'
 
+const initialState = {
```

Comment:
this is a great pattern to follow because it makes very clear what the initial state of the component should always look like. And it's protected from being mutated because it's declared with `const`